### DISCUSSION TITLE
iOS Plugin with third party library dependency

### DISCUSSION THREAD

**Discussion by kaesyar** [apache/cordova#437] [Q&A]
How to add a third party library as a dependency if it does not support cocoapods?

**Comment by breautek** [comment]
If you can package the binary with your plugin, then you should be able to use [&lt;framework&gt;](https://cordova.apache.org/docs/en/12.x/plugin_ref/spec.html#framework) tag to create a reference to it.

I have an example of what I've done for an xcframework for a custom sqlite build: https://github.com/totalpaveinc/cordova-plugin-libsqlite/blob/main/plugin.xml#L22

If you can package or build an xcframework, that will be the best thing to include since it can bundle multiple framework files (one for iOS, and the other for iOS Simulator, etc).
